### ISSUE TITLE
Can't add Yeelight manually 

### ISSUE THREAD

**Issue by yacoalioi** [rgb-master-team/RGBMaster#46]
When I enter the Yeelight IP, the device doesn't get added. 

A side note is that a logitech device is being detected, and the manual add is below the logitech tab.

**Comment by deankevorkian** [comment]
You are correct. We broke the manual IP feature a while ago and it needs a revision. Will get to it soon and hope it arrives at the upcoming version
